.github/pull-request-template.md

```diff
@@ -8,9 +8,17 @@ Fixes #
 
 **Release Note**
 
-<!-- Enter your extended release note in the below block. If the PR requires
-additional action from users switching to the new release, include the string
-"action required". If no release note is required, write "NONE". -->
+<!--
+In the following cases, write a brief release note describing the
+user-visible impact of this change in the release-note block:
```

Comment:
Can we not drop the "action required". I think that was VERY useful?

Reply:
Done.